src/mixins/PerItem.js

```diff
@@ -22,6 +22,26 @@ const PerItemMixin = types
     },
   }))
   .actions(self => ({
+    _validatePerItem() {
```

Comment:
what's the idea behind underscore here? we use underscores rarely and for purely internal things, and those are called from other models using this mixin.

Reply:
The idea was to prevent using this method outside of the bundle `PerItemMixin <-> ClassificationBase`. I'm not sure how to prevent it in better way.